server/controller.js

```diff
@@ -309,16 +367,75 @@ const handlerFunctions = {
     res.status(200).send(pantryRecipes)
   },
 
-  // get all ingredients
-
   // get all recipes
+  getAllRecipes: async (req, res) => {
+    const { pageNum } = req.params
+    const allRecipes = await Recipe.findAll({ 
+      offset: pageNum * 20, 
+      limit: 20,
+      order: [
+        ['recipeId', 'ASC']
+      ]
+    })
 
-  // get all recipe ingredients
+    console.log('allRecipes:', allRecipes)
```

Comment:
We may want to comment this out to quiet the noise in the server console. We could un-comment it if one of us is doing development with the parent function.